pocs/camera/camera.py

```diff
@@ -521,15 +510,13 @@ def command(self, cmd):
                     stdout=subprocess.PIPE,
                     stderr=subprocess.STDOUT,
                     universal_newlines=True,
-                    shell=False
-                )
+                    shell=False)
```

Comment:
I'm also not the biggest fan of putting the closing parentheses on last line as opposed to aligned with opening of call.

Reply:
This is from pep8:

When trailing commas are redundant, they are often helpful when a version control system is used, when a list of values, arguments or imported items is expected to be extended over time. The pattern is to put each value (etc.) on a line by itself, always adding a trailing comma, and add the close parenthesis/bracket/brace on the next line. However it does not make sense to have a trailing comma on the same line as the closing delimiter (except in the above case of singleton tuples).

```
Yes:

FILES = [
    'setup.cfg',
    'tox.ini',
    ]
initialize(FILES,
           error=True,
           )
```